lib/pharos/addon.rb

```diff
@@ -2,11 +2,11 @@
 
 require 'dry-validation'
 require_relative 'addons/struct'
-require_relative 'phases/logging'
+require_relative 'logging'
 
 module Pharos
   class Addon
-    include Pharos::Phases::Logging
+    include Pharos::Logging
```

Comment:
This is in the #117 too

Reply:
This is necessary here because `Pharos::Addon` uses `Pharos::Phases::Logging`, and the logging in `Pharos::Phases::Base` is customized to include the host as a prefix.